### PR TITLE
Add jumio callback support

### PR DESCRIPTION
**Pull request by NullSoldier**
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: src/redemptions/redemption.service.ts [hunk 1]
```diff
@@ -31,9 +31,38 @@ export class RedemptionService {
...
+    redemption: Redemption,
+    decisionStatus: DecisionStatus,
+    decisionLabel: DecisionLabel,
+  ): Promise<KycStatus> {
+    let returnRedemption: Redemption;
+    if (decisionStatus === DecisionStatus.NOT_EXECUTED) {
+      returnRedemption = await this.prisma.redemption.update({
+        data: { kyc_status: KycStatus.TRY_AGAIN },
+        where: { id: redemption.id },
+      });
```
Comment (jowparks) [inline src/redemptions/redemption.service.ts:44]: Work left here to do

File: src/jumio-transactions/jumio-transaction.service.ts [hunk 2]
```diff
@@ -59,6 +59,7 @@ export class JumioTransactionService {
       data: {
         decision_status: data.decisionStatus,
         last_workflow_fetch: instanceToPlain(data.transactionStatus),
+        latest_callback: new Date(),
```
Comment (NullSoldier) [inline src/jumio-transactions/jumio-transaction.service.ts:62]: latest_callback_at

File: src/jumio-kyc/kyc.service.ts [hunk 3]
```diff
@@ -112,6 +112,13 @@ export class KycService {
       return;
     }
 
+    if (
+      latest.latest_callback_at &&
```
Comment (NullSoldier) [inline src/jumio-kyc/kyc.service.ts:116]: Change this to transaction

File: src/jumio-kyc/kyc.service.ts [hunk 4]
```diff
@@ -112,6 +112,13 @@ export class KycService {
...
+      latest.latest_callback_at > new Date(data.callbackSentAt)
```
Comment (NullSoldier) [inline src/jumio-kyc/kyc.service.ts:117]: >=

**Review by jowparks — APPROVED** [review]

**Pull request merged**